monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md

```diff
@@ -0,0 +1,129 @@
+
+_**Шурэ:**_ Просто хочу отметить, что даже несмотря на то, что Bitcoin не скрывает транзакций, в прошлом предлагалось ввести конфиденциальные транзакции в рамках проекта, но без кольцевых подписей, а просто скрывать суммы, и это было оригинальное предложение по созданию конфиденциального актива, которое озвучил Грег Максвелл.
+
+_**Саранг:**_ Да, было такое предложение для Bitcoin, а затем его реализовали мы, но в форме колец.
+
+_**Джастин:**_ В нашей серии эпизодов Breaking Monero мы не касались лежащей в основе математики, мы говорили только о верхнем концептуальном уровне, чтобы люди могли понять, что происходит на самом деле, и один человек спросил нас, даже если мы не станем вдаваться в математические подробности, какие учебники лучше всего помогут ознакомиться с этими основами, чтобы узнать больше конкретно по этой теме? Итак, я снова начну с тебя, Саранг. Что бы ты порекомендовал людям, которые начинают, возможно, со студенческого уровня и хотят узнать больше об этих криптографических схемах?
+
+_**Саранг:**_ Ну, скажем так, тебе повезло, потому что совершенно случайно у меня есть с собой пара книг. Специально ли я принёс их, или они постоянно лежат у меня под рукой — вам решать.
+
+_**Шурэ:**_ Боооотан… бооооотан…
+
+_**Саранг:**_ А, это да. Итак, многое здесь зависит от уровня, с которого вы готовы начать, а также знаний, которые у вас уже имеются на текущий момент. Если вы знакомы и справляетесь с алгеброй на студенческом уровне и хотите начать с этого, то есть одна хорошая книга, вот она, книга написана Хофштейном, Пифером и Сильверманом и называется Введение в математическую криптографию. Есть издания поновее, есть постарше, в зависимости от того, что вам нужно. Конкретно это издание даёт хорошее представление о некоторых классических и современных методах шифрования. Предполагается, что для начала у вас уже имеется базовое знание алгебры уровня средней школы или колледжа. Это хорошая книга для начала. То есть с точки зрения продвинутых методов она не так хороша, равно как и для создания безопасных решений. В ней рассказывается о математике, стоящей за такими решениями, но я бы не полагался на её содержание при создании безопасного варианта реализации. Тем не менее это очень хорошая книга. Следующая книга пригодится тем, кто больше заинтересован в формальном подходе, и она не для слабонервных. Это Введение в современную криптографию Каца и Линделла. Эта книга предназначена для продвинутых студентов и начинающих аспирантов. В ней описаны формальные варианты реализации определённых конструкций. Это очень-очень хорошая и очень-очень полная книга. Я бы не советовал браться за неё новичкам. В ней очень хорошо описаны формальные варианты. Есть ещё множество хороших книг, но эти две охватывают два важных уровня. Они будут полезны тем, кто начинает разбираться с математикой и действительно хочет усилить доказательства безопасности и заняться подобными вещами.
```

Comment:
- [Introduction to Mathematical Cryptography](https://www.goodreads.com/book/show/3884171-an-introduction-to-mathematical-cryptography)
- [Introduction to Modern Cryptography](https://www.goodreads.com/book/show/24387035-introduction-to-modern-cryptography-second-edition)